src/main/java/org/mitre/synthea/engine/Module.java

```diff
@@ -276,6 +279,14 @@ public Module(JsonObject definition, boolean submodule) throws Exception {
       }
     }
 
+    if(definition.has("gmfVersion")) {
```

Comment:
Any reason you put this below the remarks processing block?

Reply:
That's a good point. I just threw it in the middle of the constructor. It should go towards the top, because there is no point in processing remarks if we are just going to throw an exception later.